inc/admin/admin.php

```diff
@@ -397,10 +397,11 @@ function rocket_analytics_data() {
 		'analytics_enabled'       => 1,
 	];
 
-	$theme              = wp_get_theme();
-	$data               = array_diff_key( get_option( WP_ROCKET_SLUG ), $untracked_wp_rocket_options );
-	$locale             = explode( '_', get_locale() );
-	$data['web_server'] = 'Unknown';
+	$theme                 = wp_get_theme();
+	$data                  = array_diff_key( get_option( WP_ROCKET_SLUG ), $untracked_wp_rocket_options );
+	$locale                = explode( '_', get_locale() );
+	$data['web_server']    = 'Unknown';
+	$data['http_protocol'] = '';
```

Comment:
I think no need to use this initialization as always the function `wp_get_server_protocol` will return valid value.